authors/apps/authentication/renderers.py

```diff
@@ -21,6 +24,17 @@ def render(self, data, media_type=None, renderer_context=None):
             rendering errors.
             """
             return super(UserJSONRenderer, self).render(data)
+        try:
```

Comment:
@Bileonaire Please comment your code for easy review

Reply:
sorted @PAbishai